script/Ciel/Day 13/13_01_01_CIEL13_5B.txt

```diff
@@ -1470,7 +1474,7 @@ Since he was the first to call out, he was the one to be swarmed and devoured.
 [sha:760ee5ac8f005416153420080c57de8beafa6f71]{
 -- Page 655, Offset 35149.
 -- わたしは広場に捨てられたまま、骨だけになっていく彼と、変わり果てた町をみあげて、みあげて―――
-I lie in the town square where I've fallen after being shoved,
+I lie discarded in the town square,
  watching as he is reduced to bones while looking up, and up and up;
  seeing with my own two eyes―――
 }
```

Comment:
35151
Just realized this line is sort taken away from its context to give fit it into a cliché English line.
She's giving a comparison.

>But that end wasn't something noteworthy.

"my town dead nooo :(" > "though, not much of anything becaauuuuseee" > "the world is going crazy!" > "we were just used for this ritual, you see"

Could maybe change the follow up to this line to reflect that causality a bit better.

Reply:
35157
Minor but I think simple past works better than perfect present
>so my town was used for this ritual.